src/pages/kb/data-sources/jira.md

```diff
@@ -9,6 +9,8 @@ slug: jira-ds
 
 You need a **Username** and **[API Token](https://confluence.atlassian.com/cloud/api-tokens-938839638.html)** to connect to Jira. The token behaves like a password. 
```

Comment:
How about:

```suggestion
You need a **Username** (the email address you use to login) and **[API Token](https://confluence.atlassian.com/cloud/api-tokens-938839638.html)** to connect to Jira. The token behaves like a password. 
```

instead of a separate line?

Reply:
Great idea.